### PR TITLE
Assessment

### PR DESCRIPTION
**Pull request by jcmeyer10**
Let me know if this works!

### CODE REVIEW

File: lib/assessment.rb [hunk 1]
```diff
@@ -2,38 +2,60 @@
...
-##
+# Create a class called Dog. It should have, as attributes, name, breed,
+# and age. Make all of those attributes read-only.
+
+# stupid rubocop error message, also, is trailing white space
+# THAT big a deal?
+class Dog
+  attr_reader :name
+  attr_reader :age
+  attr_reader :breed
```
Comment (jrhorn424) [inline lib/assessment.rb:13]: Cool. This can be done in one line, though. :smile:

File: lib/assessment.rb [hunk 2]
```diff
@@ -2,38 +2,60 @@
...
 
 # Question 3
 # Create a Person class, with attributes name, age, and location.
 # It should set all three attributes when you create a new Person.
 # Name and location should be write-only, but age should be read-only.
 
-##
-# your code here
-##
+# just want to get rid of that class error from rubocop
```
Comment (jrhorn424) [inline lib/assessment.rb:32]: We should setup rubo-cop for you. Will you file an issue in `meta`?

File: lib/assessment.rb [hunk 3]
```diff
@@ -2,38 +2,60 @@
...
+  end
+end
 
 # Question 4
 # Create a new Person object using parameters "Dave", 32, and "Ohio".
 # Then, set that Person's location to "Somerville"
 
-Answer.dave = nil # replace nil with your code
-Answer.dave.location = nil # replace nil with your code
+Answer.dave = Person.new('Dave', 32, 'Ohio') # replace nil with your code
```
Comment (jrhorn424) [inline lib/assessment.rb:49]: What gets set as `@age` when you do this? Look at line 38.

**Comment by jrhorn424** [comment]
Did you run the tests? It would have caught the mistake you made on question 3 (or question 4, depending on your point-of-view).

**Comment by jrhorn424** [comment]
Good work! 4.5/5

**Comment by jcmeyer10** [comment]
Thanks man.  I do have RuboCop and it seemed to check it all out before
submitting.  That's how I learned that ' ' are used when there is no reason
to use " " (ie #{variable}).

On Mon, Nov 2, 2015 at 4:27 PM, Jeffrey Horn notifications@github.com
wrote:

> Good work! 4.5/5
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ga-wdi-boston/ruby-objects-assessment/pull/51#issuecomment-153162593
> .

## 

Regards,
Jeff Meyer

857-206-3572

**Comment by jrhorn424** [comment]
Did you see your error in question 3?

**Pull request closed without merging**